### ISSUE TITLE
This line should be corrected.

### ISSUE THREAD

**Issue by deeperlearner** [fdbtrs/Self-restrained-Triplet-Loss#12]
https://github.com/fdbtrs/Self-restrained-Triplet-Loss/blob/adb9421fb8171b530d1351a59763c3be6f0fd346/main.py#L141

It should be
`cnn_optimizer.zero_grad() `
